jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jooq</groupId>
     <artifactId>jooq-spring-boot-example</artifactId>
-    <version>${org.jooq.version}</version>
+    <version>${jooq.version}</version>
```

Comment:
Thank you very much for your help. That must've been my misunderstanding then, or my lack of maven-fu. I can see it now. Will restore the commit.